pyteal/ast/router.py

```diff
@@ -63,19 +63,6 @@ def approval_condition_under_config(self) -> Expr | int:
             case _:
                 raise TealInternalError(f"unexpected CallConfig {self}")
 
-    def clear_state_condition_under_config(self) -> int:
```

Comment:
I'd like to see tests that confirm an error is raised if you attempt to provide a clear state bare app call or method.